src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/scenarios/manageCaseAssignments/casesharing.scala

```diff
@@ -0,0 +1,259 @@
+      .post(mcaUrl + "/case-assignments")
+      .header("ServiceAuthorization", "Bearer ${bearerTokenShare}")
+      .header("Authorization", "Bearer ${access_tokenShare}")
+      .header("Content-Type", "application/json")
+      .header("Accept", "application/json")
+      .body(StringBody("{\"assignee_id\":\"${assigneeUser}\",\"case_id\":\"${caseToShare}\",\"case_type_id\":\"GrantOfRepresentation\"}")))
+
+      .pause(Environment.constantthinkTime)
+
+  val CaseSharingPostSmall =
+
+    exec(http("02_CS_010_AssignCaseSmallOrg")
+      .post(mcaUrl + "/case-assignments")
+      .header("ServiceAuthorization", "Bearer ${bearerTokenShare}")
+      .header("Authorization", "Bearer ${access_tokenShare}")
+      .header("Content-Type", "application/json")
+      .header("Accept", "application/json")
+      .body(StringBody("{\"assignee_id\":\"${assigneeUser}\",\"case_id\":\"${caseToShare}\",\"case_type_id\":\"GrantOfRepresentation\"}")))
+
+      .pause(Environment.constantthinkTime)
+
+  val NoticeOfChangeRequest =
+
+    exec(http("NoC_010_ApplyDecision")
+      .post(mcaUrl + "/noc/apply-decision")
+      .header("ServiceAuthorization", "Bearer ${bearerTokenShare}")
+      .header("Authorization", "Bearer ${access_tokenShare}")
+      .header("Content-Type", "application/json")
+      .header("Accept", "application/json")
+      .body(StringBody("{\n\t\"case_details\": {\n\t\t\"reference\": \"1601639930193327\",\n\t\t\"data\": {\n\t\t    \"DateField\": null,\n\t\t    \"TextField\": \"TextFieldValue\",\n\t\t    \"EmailField\": \"aca72@gmail.com\",\n\t\t    \"NumberField\": \"123\",\n\t\t    \"CollectionField\": [],\n\t\t    \"FixedRadioListField\": null,\n\t\t    \"MultiSelectListField\": [],\n\t\t    \"OrganisationPolicyField1\": {\n\t\t        \"Organisation\": {\n\t\t            \"OrganisationID\": \"AddingOrg\",\n\t\t            \"OrganisationName\": null\n\t\t        },\n\t\t        \"OrgPolicyReference\": \"DefendantPolicy\",\n\t\t        \"OrgPolicyCaseAssignedRole\": \"[Defendant]\"\n\t\t    },\n\t\t    \"OrganisationPolicyField2\": {\n\t\t        \"Organisation\": {\n\t\t            \"OrganisationID\": null,\n\t\t            \"OrganisationName\": null\n\t\t        },\n\t\t        \"OrgPolicyReference\": \"ClaimantPolicy\",\n\t\t        \"OrgPolicyCaseAssignedRole\": \"[Claimant]\"\n\t\t    },\n\t\t    \"ChangeOrganisationRequestField\": {\n\t\t        \"Reason\": null,\n\t\t        \"CaseRoleId\": \"[Defendant]\",\n\t\t        \"NotesReason\": \"Some notes\",\n\t\t        \"ApprovalStatus\": 1,\n\t\t        \"RequestTimestamp\": null,\n\t\t        \"OrganisationToAdd\": {\n\t\t        \t  \"OrganisationID\": \"AddingOrg\",\n\t\t            \"OrganisationName\": null\n\t\t        },\n\t\t        \"OrganisationToRemove\": {\n\t\t\t\t  \"OrganisationID\": \"QUK822N\",\n\t\t            \"OrganisationName\": null\n\t\t        },\n\t\t        \"ApprovalRejectionTimestamp\": null\n\t\t    }\n\t\t}\n\t}\n}")))
```

Comment:
Is this meant to have hardcoded data in (such as a case number)? Could this be better placed in a JSON file?

Reply:
this code is existing in the repo and not mine - the casesharing.scala file has only been moved into a new folder.

Reply:
I have now removed this file from the repo as the code is being rewritten as part of the manage case PI28